jib-core/src/main/java/com/google/cloud/tools/jib/json/JsonTemplateMapper.java

```diff
@@ -69,10 +70,16 @@
 
   /** Convert a {@link JsonTemplate} to a {@link Blob} of the JSON string. */
   public static Blob toBlob(JsonTemplate template) {
-    return Blobs.from(
-        outputStream -> {
-          objectMapper.writeValue(outputStream, template);
-        });
+    return mapToBlob(template);
+  }
+
+  /** Convert a list of {@link JsonTemplate}s to a {@link Blob} of the JSON string. */
+  public static Blob toBlob(List<JsonTemplate> template) {
+    return mapToBlob(template);
+  }
+
+  private static Blob mapToBlob(Object template) {
```

Comment:
This can actually be called `toBlob` as well since I believe Java resolves to the strictest type match. (The calls to it would need to be typecasted to Object though)